pyproject.toml

```diff
@@ -38,6 +38,12 @@ pyright = "^0.0.12"
 pre-commit = "^2.15.0"
 pytest-cov = "^3.0.0"
 setuptools-rust = "^1.1.2"
+
```

Comment:
You probably need to use `[tool.poetry.group.dev.dependencies]` above to make our use of group consistent (and make the Nix build pass).

Reply:
```diff
diff --git a/pyproject.toml b/pyproject.toml
index 0567094..f8bfb64 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -31,8 +31,10 @@ lark = "^1.1.2"
 lz4 = "^4.0.0"
 lief = "^0.12.3"
 
+[tool.poetry.group.dev]
+optional = true
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
 pytest = "^6.2.4"
 pyright = "^0.0.12"
 pre-commit = "^2.15.0"
```

Reply:
Good catch, moved it.